packages/video_player/lib/video_player.dart

```diff
@@ -230,6 +237,16 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
             buffered: values.map<DurationRange>(toDurationRange).toList(),
           );
           break;
+        case 'bufferingStart':
+          value = value.copyWith(
+            isBuffering: true,
+          );
```

Comment:
[nit] Here and below: we use trailing commas only when needed. And this statement easily fits on a single line:
```dart
value = value.copyWith(isBuffering: true);
```

Reply:
Ah yeah sorry, force of habit. Will remove.